bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,862 @@
+| Services Supported | LastDestinations    | destinations                 |
+| Services Activated | list of services    | remote                       |
+
+
+## Discovery
+
+Auto Discovery is starting after creation of the Bridge towards BMW ConnectedDrive. 
+A list of your registered Vehicles is queried and all found Vehicles are added in Inbox.
+Unique Identifier is the *Vehicle Identification Number* (VIN). 
+If a Thing is already declared in a  *.things configuration* Discovery won't highlight it again.
+Properties will be attached to predefined Vehicles if the VIN is matching.
+
+## Configuration
+
+### Bridge
+
+| Parameter       | Type    | Description                                                             |           
+|-----------------|---------|-------------------------------------------------------------------------|
+| userName        | text    | BMW Connected Drive Username                  |
+| password        | text    | BMW Connected Drive Password                  |
+| region          | text    | Select Region in order to connect to the appropriate BMW Server.   |
+
+The region Configuration has 3 different possibilities
+
+* _NORTH_AMERICA_
+* _CHINA_
+* _ROW_ for Rest of World
+
+### Thing
+
+Same Configuration is needed for all Things
+
+| Parameter       | Type    | Description                           |           
+|-----------------|---------|---------------------------------------|
+| vin             | text    | Vehicle Identification Number (VIN)   |
+| refreshInterval | integer | Refresh Interval in Minutes           |
+| units           | text    | Unit Selection                        |
+| imageSize       | integer | Image Size                            |
+| imageViewport   | text    | Image Viewport                        |
+
+The units can be configured in 3 ways
+
+* _AUTODETECT_ selects Miles for US & UK, Kilometer otherwise
+* _METRIC_ selects directly Kilometers
+* _IMPERIAL_ selects directly Miles
+
+The imageVieport allows to show the Vehicle from different angels.
+Possible values are 
+
+* FRONT
+* REAR
+* SIDE
+* DASHBOARD
+* DRIVERDOOR
+
+## Channels
+
+There are many Channels available for each Vehicle. 
+For better overview they are clustered in different Channel Groups.
+The Channel Groups are different for the Vehicle Types, on the build-in Sensors of the Vehicle and the activated Services.
+
+### Bridge Channels
+
+If the Vehicle isn't found in the Discovery force a log of Fingerprint Data which helps to analyze the problem.
+Please check [TroubleShooting Section](#TroubleShooting) for further advice.
+
+| Channel Label         | Channel ID            | Type   | Description                                       |
+|-----------------------|-----------------------|--------|---------------------------------------------------|
+| Discovery Fingerprint | discovery-fingerprint | Switch | Forcing a log entry to analyze Discovery Problems |
```

Comment:
Logging repeatedly some errors e.g. when retrieving _wrong_ information is not wanted. So I took over the [fingerprint concept from BimmerConnected](https://github.com/bimmerconnected/bimmer_connected#data-contributions). With this it can be checked if it's a server issue, authorization issue or a logical problem in the code. In the latter case unit test will be expanded with the new data.

Reply:
Still, I agree with @fwolter that there should not be channels to adapt logging behavior.
If the user wants to debug some issues, he should switch the binding to debug logging and find in there what he needs.

Reply:
Fingerprint is now logged every time on startup / thing initialization instead of user command.